README.md

```diff
@@ -27,6 +27,11 @@ If you want to run it locally
   - The page uses Jekyll's `site.github.public_repositories` object to get the repo data. This data is only available when live.
   - Custom repo data is generated by Github's JS API, so it will show up locally.
 
+### Content Management
+- The content for each project comes from the GitHub Pages for that repo.
+This can be from a `gh-pages` branch, or from the `/docs` directory, based on the GitHub Pages settings.
+Using the `gh-pages` branch appears to be deprecated by GitHub, and can be confusing for contributors.
```

Comment:
Did they announce this somewhere? (Great if so! Using a gh-pages branch is so weird.)

Reply:
@tanema had mentioned it [here](https://github.com/Shopify/shopify_api/pull/734#issuecomment-634750466) but I couldn't find anything definitive, and it's still in the docs, so may be better to remove?

Reply:
I think we should definitely say the `/docs` method is preferred because it's less confusing for contributors — maybe just omit the part about GitHub deprecating it. (It does seem like they don't highlight it as much as they used to, but it's a bit unclear.)

Reply:
Ok, fixed in aae56d0ba7e5ff7e6716a917918e70cb3dfd7469.